.github/workflows/canary.yml

```diff
@@ -33,10 +33,10 @@ jobs:
         uses: actions/cache@v2
         with:
           path: '**/node_modules'
-          key: ${{runner.os}}-16-8-16-node-modules-${{hashFiles('yarn.lock')}}
+          key: ${{runner.os}}-yarn-${{hashFiles('yarn.lock')}}
           restore-keys: |
-            ${{runner.os}}-16-8-16-node-modules-${{hashFiles('yarn.lock')}}
-            ${{runner.os}}-16-8-16-node-modules-
```

Comment:
I think we should respect versions/variations on tests (for Nodejs version matrix etc)

Reply:
yeah, will change